bin/tx-prover/src/commands/mod.rs

```diff
@@ -47,6 +56,45 @@ impl Default for ProxyConfig {
     }
 }
 
+impl ProxyConfig {
+    /// Loads config file from current directory and default filename and returns it
+    ///
+    /// This function will look for the configuration file with the name defined at the
+    /// [PROVER_SERVICE_CONFIG_FILE_NAME] constant in the current directory.
+    pub(crate) fn load_config_from_file() -> Result<ProxyConfig, String> {
+        let mut current_dir = std::env::current_dir().map_err(|err| err.to_string())?;
+        current_dir.push(PROVER_SERVICE_CONFIG_FILE_NAME);
+        let config_path = current_dir.as_path();
+
+        Figment::from(Toml::file(config_path))
+            .extract()
+            .map_err(|err| format!("Failed to load {} config file: {err}", config_path.display()))
+    }
+
+    pub(crate) fn save_to_config_file(&self) -> Result<(), String> {
```

Comment:
Let's add doc comments to this function.

Reply:
Just added it ✅ .